add-ons/dspace-docker-tools/ingestAIP.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# AIPZIP=${AIPZIP:-https://github.com/DSpace-Labs/AIP-Files/raw/master/DogPhotosAIP.zip}
```

Comment:
We is this commented out and set later? While this can be set as an enviornment variable, I would consider it helpful to have this quite at the begining of the script. Maybe we should add a note that it can be changed by an environment variable.

Reply:
Code updated